### DISCUSSION TITLE
taro组件能否集成到create react app脚手架提供的工程里吗？

### DISCUSSION THREAD

**Discussion by blue3xp** [NervJS/taro#13879] [Q & A] [answered]
### 这个特性解决了什么问题？
可以使用taro编写公共ui组件，并能够集成在react工程里吗

### 这个 API 长什么样？
有一个脚手架工程提供

**Comment by ZakaryCode** [comment, marked as answer]
可以，Web 环境接入 taro 组件可以使用[组件编译模式](https://nervjs.github.io/taro-docs/docs/h5/#%E7%BB%84%E4%BB%B6%E7%BC%96%E8%AF%91%E6%A8%A1%E5%BC%8F)

**Reply by ZakaryCode** [reply]
> 有一个脚手架工程提供

在前端 UI 框架中使用，可以自行引入编译好的组件即可，无需提供脚手架

**Reply by blue3xp** [reply]
我采用的是另一种方式 通过改webpack的配置，可以让taro的代码运行在nextjs和create react app的工程里面，demo代码是
https://github.com/blue3xp/taro-create-react-app
https://github.com/blue3xp/taro-nextjs

**Reply by ccclllll** [reply]
> 我采用的是另一种方式 通过改webpack的配置，可以让taro的代码运行在nextjs和create react app的工程里面，demo代码是 https://github.com/blue3xp/taro-create-react-app https://github.com/blue3xp/taro-nextjs

有办法在项目里使用taro-ui，nut-ui-taro吗
